go.mod

```diff
@@ -1,4 +1,4 @@
-module golift.io/starr
+module github.com/jackdallas/starr
```

Comment:
You should use `replace =>` in your `go.mod` instead of doing this. Can't merge this as-is.

Reply:
Hey my bad I meant to push this to another branch temporarily, cleaned up :+1: